### PR TITLE
Added post about our brand

### PR DESCRIPTION
**Pull request by davidaayers**

### CODE REVIEW

**Comment by davidaayers** [comment]
I'd like 3 reviews please 😄

**Comment by twoqubed** [comment]
Would it be too cheesy to bold a few key words in each principle? For example...

* We exhibit a __bias to action__, and prefer those things that move us forward; everything we do should be in the service of getting something done.
* We encourage __curiosity and exploration__, but always with an eye towards pragmatic implementation, as we must support and maintain the things we build

**Comment by jrdaniels** [comment]
I like Ryan's idea; otherwise 👍

**Comment by davidaayers** [comment]
No, not too cheesy, I was actually thinking of something similar. I'll take a crack at it.

**Comment by davidaayers** [comment]
@twoqubed & @jrdaniels How's this look?

**Comment by jrdaniels** [comment]
I like it.  It makes me think that eventually each of the bolded items could be a post of it's own, with this page linking to the post.

**Comment by scrozier** [comment]
👍

File: _posts/2016-11-08-ourbrand.markdown [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+---
+# A Brief Side Trip
+
+For those of you excited about the next post in my agility series, I'm afraid you'll be disappointed. I'm going to take a brief trip sideways, and come back to the agility series next week. But I'm hoping after reading this your disappointment will abate and you'll be as excited as I am.
+
+# Creating A Brand
+
+Lots of folks in John T's area with direct reports have been meeting recently on the topic of being better managers. These have been really great -- I think it's easy for everyone to overlook the value of these "soft skills", especially those of us on the technical side of things. Over the last several years, I have really begun to appreciate how important and impactful they can be.
+
+In one of these meetings, John challenges everyone to come up with a "brand" for their area. What are those things that we *stand for*. Not to wax too philosophical, but it took me on a bit of a soul search. What did we stand for, as a department? What did we want other people to think when they thought of us? And more importantly, how do we achieve this lofty vision?
```
Comment (JohnThrailkill) [inline _posts/2016-11-08-ourbrand.markdown]: should be challenged, not challenges

File: _posts/2016-11-08-ourbrand.markdown [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+* We care deeply about the voice of our customer, and strive to include that voice whenever possible by (at a minimum) incorporating feedback loops and analytics into everything we build.
+
+It's important to note that some of these principles are inspirational; we may not be hitting the mark in that particular area, but we are striving to make things better.
+
+I've incorporated our brand statement into the blog, if you look at the top right navigation, you'll see a link that takes you to a page dedicated to it.
+
+# Back To Our Regularly Scheduled Program
+
+Thanks for indulging me this week; I'm exciting about sharing our brand and working toward realizing it with all of our stakeholders. We'll return next week with the next article in our agility series, "Roles in the Scrum Process".
```
Comment (JohnThrailkill) [inline _posts/2016-11-08-ourbrand.markdown]: excited, not exciting

**Comment by JohnThrailkill** [comment]
Looks great - I just commented on two typos.  Good stuff!

**Comment by edwagner** [comment]
Great content and great idea to publish our brand. I sent you a couple of trivial suggestions in email.

**Comment by JohnThrailkill** [comment]
Email?! Do you need some GitHub training, Ed? ☺

John Thrailkill | Executive Vice President of IT & Business Development | The Container Store
972-538-6649 phone | 972-538-7649 fax
containerstore.com<http://www.containerstore.com/#containerstore.com> | [cid:image006.jpg@01D23AAC.6A0C42A0] <http://www.pinterest.com/ContainerStore>   [cid:image007.jpg@01D23AAC.6A0C42A0] <http://www.facebook.com/containerstore>   [cid:image008.jpg@01D23AAC.6A0C42A0] <http://www.twitter.com/ContainerStore>   [cid:image009.jpg@01D23AAC.6A0C42A0] <http://instagram.com/thecontainerstore>
[cid:image010.jpg@01D23AAC.6A0C42A0]<http://standfor.containerstore.com/> What We Stand For<http://standfor.containerstore.com/#What%20We%20Stand%20For>

**Pull request merged**